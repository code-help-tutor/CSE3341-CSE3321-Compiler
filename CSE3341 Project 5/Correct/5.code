procedure t5 and
	procedure A(x) is
		record y;
	begin
		y := new record[1]; 
		y[0] := 8; 
		out(x); 
		x := record y; 
	end
is
	integer m;
	integer n;
	record z;
begin
	m := 10;
	n := 5;
	z := new record[1];
	z[0] := m+n;
	begin A(z);
	out(m);
end
