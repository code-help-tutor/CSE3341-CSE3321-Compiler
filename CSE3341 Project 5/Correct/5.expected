gc:1
gc:2
15
gc:1
10
gc:0
